### ISSUE TITLE
Tasks:  completed tasks have a submission on V4, but not on V5

### ISSUE THREAD

**Issue by SathyaJayabal** [topcoder-platform/challenge-api#269]
Tasks:  completed tasks have a submission on V4, but not on V5. 

@Oanh-and-only-Oanh , not sure if this is an issue.

example:
<img width="1440" alt="Screenshot 2020-08-13 at 1 36 25 PM" src="https://user-images.githubusercontent.com/58783823/90111067-bcfa9780-dd6b-11ea-9366-42be1675708b.png">
<img width="1440" alt="Screenshot 2020-08-13 at 1 48 36 PM" src="https://user-images.githubusercontent.com/58783823/90111076-c126b500-dd6b-11ea-8d8e-af9f7ec30bae.png">

**Comment by ThomasKranitsas** [comment]
AFAIK, on v4, we create some dummy submission record in order to be able to close the challenge. I'm not sure if that's what's causing this issue.

**Comment by SathyaJayabal** [comment]
Yes, you are right. If @Oanh-and-only-Oanh confirms if its ok, we can close this issue.

**Comment by SathyaJayabal** [comment]
> AFAIK, on v4, we create some dummy submission record in order to be able to close the challenge. I'm not sure if that's what's causing this issue.

@ThomasKranitsas , do you know from where the dummy submission is created?  Will this issue cause problems in closing tasks after V5 release.
cc @rootelement

**Comment by Oanh-and-only-Oanh** [comment]
From a front-end perspective, that's fine not to have submission for completed tasks. Just want to make sure we're still able to close out the challenge?

**Comment by ThomasKranitsas** [comment]
The V5 Challenge API does not require a submission to close out a challenge/task. It simply expects the winner(s) to be defined. We can close this issue as it's not a valid bug and it also doesn't make sense to require a dummy submission in order to close out a challenge/task.
